specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -1203,17 +1217,15 @@ components:
       properties:
         typeGerechtigde:
           type: "string"
-          description: "Het type zakelijk recht dat deze gerechtigde heeft. Komt overeen met AardzakelijkRecht"
+          description: "Het type zakelijk recht dat deze gerechtigde heeft."
```

Comment:
Wat is dat type? Is dit "aard" uit het imKad? 
Is dit dus eigenlijk een waardelijst of inderdaad een willekeurige string?

Reply:
Dit is een type van de zakelijkgerechtigde. Dat is een lijstje dat op ons verzoek is opgesteld en afgezegend is (of moet worden) door de bewaarder van het Kadaster. We gebruiken vooralsnog volgens mij alleen nog het type "eigenaar". Lijkt mij dat dit een waardelijst moet worden, maar dat volgt uit de borging van die lijst ls hij wordt goedgekeurd. (Neem ik aan)

Reply:
Ter aanvulling: Overigens wordt deze property ook als parameter gebruikt en daar staat als reminder met vette letters dat de waardelijst nog moet worden toegevoegd.

Reply:
> Dit is een type van de zakelijkgerechtigde. Dat is een lijstje dat op ons verzoek is opgesteld en afgezegend is (of moet worden) door de bewaarder van het Kadaster. We gebruiken vooralsnog volgens mij alleen nog het type "eigenaar". Lijkt mij dat dit een waardelijst moet worden, maar dat volgt uit de borging van die lijst ls hij wordt goedgekeurd. (Neem ik aan).

Is goedgekeurd!